enterprise/bundle.go

```diff
@@ -7,14 +7,9 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/fileresolver"
 )
 
-// NB: We cannot embed "static", "bazel-out", etc here, because when this
-// package is built as dependency of the enterprise package, those files
-// do not exist. Instead we bundle *, which never fails, although the
-// resulting filesystem may be empty.
-//
-//go:embed *
+//go:embed LICENSE
```

Comment:
does this need to be "*"? If not, I'm a bit confused on how the `config_files` are bundled here :thinking:

Reply:
They aren't, they're bundled in `enterprise/config/bundle.go`.